styleguide.md

```diff
@@ -130,3 +138,10 @@ Not every single page or step needs an accompanying screenshot. When in doubt, f
 ### File format and naming:
 
 Please use .png, .gif, or .jpg files. Give the screenshots unique, meaningful names, obeying a specific format, such as general_settings_privacy. Avoid spaces, uppercase letters, and excessively long names.
+
+### Alt text:
+
+As a best practice, all screenshots should have accompanying alt text. 
+
+Use the following format when inserting an image into the docs:
+
```

Comment:
I couldn't find an actual screenshot/image with alt text (oops). Can you help me figure out the right format please and thank you?

Reply:
We include the brackets for it, we just don't currently populate it. Oops.
So here's our current text expander image:
`![]({{root_url}}/images/<image_name>.png "Alt text")`
and here's what it should be with alt text:
`![alt text here]({{root_url}}/images/<image_name>.png)`